website/pages/docs/autoscaling/internals/checks.mdx

```diff
@@ -16,12 +16,12 @@ retaining the most capacity of the resource.
 In a scenario where two checks return different desired directions, the following
 logic is applied.
 
-- `ScaleOut and ScaleIn   => ScaleOut`
+- `ScaleOut and ScaleIn => ScaleOut`
```

Comment:
Note: These appear to be attempts at lining up the output vertically, but they actually don't get rendered this way: https://www.nomadproject.io/docs/autoscaling/internals/checks.

Reply:
This change is undesirable, imo. Removing the spaces here would be a bug and lost behavior from the [commonmark spec (example 337)](https://spec.commonmark.org/0.29/#example-337).

As the spec suggests, adding `white-space: pre-wrap` to inline code elements will preserve spaces. I tried it:

![image](https://user-images.githubusercontent.com/174740/94836602-03618f80-03c8-11eb-95e2-dc2fa0222fb9.png)

Reply:
@jescalan this seems like it might require a change to our MDX parsing, and our auto-formatting config across all docs sites? Also curious if this something that we've come across before, since it isn't a regression from the current version of the site.

Reply:
I have not seen this before - we don't have any custom written parsing or formatting config, it's simply prettier. It might require filing an issue with prettier, though historically this has not been wildly successful.

Reply:
Yeah, here's a one-liner that demonstrates the bug.

```shellsession
$ echo '`Some    code      here`' | npx prettier --stdin-filepath file.md
```

Although this is a prettier bug, I'm concerned if we move forward with this, we'll lose the formatting in the event that Prettier does eventually address this.

Reply:
Thank you! Really appreciate the dig and followup 🙏🏼

Reply:
So to move forward while we wait on Prettier, I can revert these few line changes, but we've set ourselves up in a difficult spot where future changes to the file will try to remove the spaces again. Is this an acceptable trade-off for now?

We could also disable the autoformatting git hook, so that it is not enforced by accident, or treat this content as a special case that is either ignored by Prettier or written in a more explicit way.

Reply:
My $0.02 is proceeding is fine as long as we don't lose track of these small things. If this is the only one, then this PR serves as a sort of record. If there are others, maybe we should stick a comment somewhere, or append the commit message.

Reply:
Ok, that makes sense to me. I just did another scan and did not find any other cases of this, so I think this thread is sufficient for posterity.